src/gui/instrument/EnvelopeGraph.cpp

```diff
@@ -54,102 +70,123 @@ void EnvelopeGraph::modelChanged()
 
 void EnvelopeGraph::mousePressEvent(QMouseEvent* me)
 {
-	if(me->button() == Qt::LeftButton)
+	if (me->button() == Qt::LeftButton)
```

Comment:
This looks like a prime short if we can collapse.

Reply:
Do you mean the following?
```
if (me->button() == Qt::LeftButton) { toggleAmountModel(); }
```

Reply:
Yep